src/main/java/hlf/java/rest/client/controller/ConfigurationController.java

```diff
@@ -0,0 +1,27 @@
+package hlf.java.rest.client.controller;
+
+import hlf.java.rest.client.model.FileUploadHandler;
+import hlf.java.rest.client.service.ControllerService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.io.IOException;
+
+@Slf4j
+@RestController
+@RequestMapping("/configuration")
+public class ConfigurationController {
+  @Autowired ControllerService controllerService;
+  @PostMapping("/update/config-file")
```

Comment:
[nitpick] Can the API be named `/update/config` instead of `/update/config-file`? It becomes too verbose for the name otherwise.